src/templates/common/encoding/article_bibtex.bib

```diff
@@ -1,7 +1,7 @@
 {% load settings %}
 {% load encoding %}
-{% if article.journal.is_conference %}@conference{% else %}@article{% endif %}{% templatetag openbrace %}{{ article.journal.code }} {{ article.id }},
-	author = {{ article.author_list|latex_conform }},
```

Comment:
Line 3 has me puzzled, could well be my lack of understanding, but here's my observation.  Please ignore if it a misunderstanding or not relevant:

I read this as a change from an output described by `templatetag openbrace journal.article.code articleid,` to `templatetag openbrace journalarticle.codearticleid,` -> i.e. that the last two terms have been concatenated because the space between them has been removed.

Was that intended?

Reply:
marking as changes requested only in as far as asking you to consider this observation before merging.

Reply:
Yes, the space makes thie field invalid.